lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -256,47 +264,56 @@ public function render_auto_login_method( $args = array() ) {
 	 * @see add_settings_field()
 	 */
 	public function render_mfa( $args = array() ) {
-		$this->render_switch( $args['label_for'], $args['opt_name'] );
 		$this->render_field_description(
-			__( 'Mark this if you want to enable multifactor authentication with Auth0 Guardian. ', 'wp-auth0' ) .
-			sprintf(
-				__( 'You can enable other MFA providers in the %s. ', 'wp-auth0' ),
-				$this->get_dashboard_link( 'multifactor' )
-			) . __( 'For more information, see our ', 'wp-auth0' ) .
+			__( 'MFA is a method of verifying identity by requiring more than 1 piece of identifying information. ', 'wp-auth0' ) .
```

Comment:
We'll keep the MFA prompt here, regardless of whether the feature is on or not.

Reply:
I found this particular line confusing. For me it is clearer to say "MFA is a method to verify a user's identity by checking a second factor in addition to the password". The phrases below are ok!